src/protagonist/API/Features/Image/Validation/HydraImageValidator.cs

```diff
@@ -28,29 +28,26 @@ public HydraImageValidator(IOptions<ApiSettings> apiSettings)
             RuleFor(a => a.MediaType).NotEmpty().WithMessage("Media type must be specified");
         });
         
-        When(a => !a.WcDeliveryChannels.IsNullOrEmpty(), DeliveryChannelDependantValidation)
-            .Otherwise(() =>
-            {
-                RuleFor(a => a.Width).Empty().WithMessage("Should not include width");
-                RuleFor(a => a.Height).Empty().WithMessage("Should not include height");
-                RuleFor(a => a.Duration).Empty().WithMessage("Should not include duration");
-            });
-        
         When(a => !a.DeliveryChannels.IsNullOrEmpty(), ImageDeliveryChannelDependantValidation);
         
+        // Legacy policy fields
+        RuleFor(a => a.ImageOptimisationPolicy)
+            .Null()
+            .When(_ => !apiSettings.Value.EmulateOldDeliveryChannelProperties)
+            .WithMessage("'ImageOptimisationPolicy' is disabled");
+
+        RuleFor(a => a.ThumbnailPolicy)
+            .Null()
+            .When(_ => !apiSettings.Value.EmulateOldDeliveryChannelProperties)
+            .WithMessage("'ThumbnailPolicy' is disabled");
+        
         // System edited fields
+        RuleFor(a => a.Width).Empty().WithMessage("Should not include width");
+        RuleFor(a => a.Height).Empty().WithMessage("Should not include height");
+        RuleFor(a => a.Duration).Empty().WithMessage("Should not include duration");

```

Comment:
are these oversimplified compared to the rules they are replacing? Taking Duration as an example the deleted rule was

```cs
RuleFor(a => a.Duration)
    .Empty()
    .WithMessage("Should not include duration")
    .Unless(a =>
        a.WcDeliveryChannels.ContainsOnly(AssetDeliveryChannels.File) && !MIMEHelper.IsImage(a.MediaType));
```

(ie you can set Duration if it only has "File" channel and it's not an image) but it's been replaced with 
```cs
RuleFor(a => a.Duration).Empty().WithMessage("Should not include duration");
```

Width, Height and Duration are set by system when ingesting is completed but if I want to serve an Audio file via the "file" channel, no transcoding has taken place so we don't know the Duration; in that case I might want to set it directly via the API.